Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -151,13 +147,16 @@ namespace gdjs {
       // Start the tween and set the needed callbacks
       tweenable
         .tween(tweenConfig)
-        .then(() => {
-          if (this._tweens[identifier])
-            this._tweens[identifier].hasFinished = true;
+        .then(
+          () => {
+            if (this._tweens[identifier])
+              this._tweens[identifier].hasFinished = true;
 
-          if (destroyObjectWhenFinished)
-            this.owner.deleteFromScene(this._runtimeScene);
-        })
+            if (destroyObjectWhenFinished)
+              this.owner.deleteFromScene(this._runtimeScene);
+          },
+          () => {}
```

Comment:
Any particular reason for this change?

Reply:
I updated the typescript type definitions to ones based on the new official ones, and it is now complaining that this second argument is not optional